Yoast/javascript
# FRO-95 Add namespace to CSS variables

## Summary


This PR can be summarized in the following changelog entry:

* [@yoast/components] Fixes a bug where our global CSS variables could conflict with global CSS variables of themes

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Pull this branch
* Pull the release/14.3 branch of the wordpress-seo repo
* Link the monorepo and build the plugin
* Check if the styling still is correct of the plugin

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
